src/main/java/com/example/webapp/Repository/DepartmentRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.example.webapp.Repository;
+
+import com.example.webapp.Model.Department;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+@Repository
+public interface DepartmentRepository extends JpaRepository<Department, Long> {
```

Comment:
I think here JpaRepository is not mandatory , you can use CrudRepository because your aplication have only CRUD features